pkgs/servers/roon-server/default.nix

```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation {
 
   src = fetchurl {
     url = "https://download.roonlabs.com/updates/production/RoonServer_linuxx64_${urlVersion}.tar.bz2";
-    hash = "sha256-TTewFN2VsBcoGwBwZ4wWRQd68nF0sfNHAcAOiyK/bd8=";
+    hash = "sha256:1v8w7nlma233xlagzawvrk5kmb50mw8g05cyz8xa2a1qsv7a8hsv";
```

Comment:
Is this digest format okay? It previously started with `sha256-` and looked like base64, and I copied this from the error output when building with the previous value, but the error output showed both the previous value and the new value in this other format.

Reply:
That's a bit weird, what version of Nix are you using?

Reply:
I'm using Nix 22.11.

Anyway, I just found this [in the wiki](https://nixos.wiki/wiki/Nix_Hash#Usage), so I'll change this to an SRI hash:

> If you find a hash that uses colon as a separator (<type>:<hash>), don't use that. This relies on undocumented behavior and is not officially supported.

Looks like they also just released another new version of Roon Server (product version 2.0.5), so I'll update that in this MR too.